### PR TITLE
Add humsarika issue #1473

### LINKED ISSUE #1473: Sarika Kushwaha
## Profile Screenshot

image
image
image

(https://github.com/humsarika)

### PR DESCRIPTION
**Pull request by humsarika**
resolves #1473 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/sarika.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://github.com/humsarika",
```
Comment (kaiwalyakoparkar) [inline profiles/sarika.json:2]: kindly add `.png` infront of the link to get your profile image
```diff
- https://github.com/humsarika
+ https://github.com/humsarika.png
```

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: profiles/sarika.json (hunk 1 above)
Comment (humsarika) [reply profiles/sarika.json:2]: I was trying to add a hosted image like others but couldn't able to do so. If I add  .png that will open my profile image not screenshot of my GitHub profile. So can I add this ---> "https://ibb.co/zFR8Hcb" link to image URL

Comment (Pradumnasaraf) [reply profiles/sarika.json:2]: Hey @kaiwalyakoparkar here we need screenshot URL not the profile picture.

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

I recommend using this image. The link is working every time

Screenshot 2022-10-07 at 10 24 55 PM

**Comment by humsarika** [comment]
@Pradumnasaraf  thank you for your help.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**